exercises/anagram/test.ml

```diff
@@ -4,7 +4,7 @@ open Anagram
 
 let ae exp got _test_ctxt =
   let printer = String.concat ~sep:";" in
-  assert_equal exp got ~cmp:(List.equal ~equal:String.equal) ~printer
+  assert_equal exp got ~cmp:(List.equal String.equal) ~printer
```

Comment:
```suggestion
  assert_equal exp got ~printer
```

Hi, and thanks for your PR!

It seems that this commit breaks CI in a way that relates to the issue you highlight.

Even though I'm not a fan of implicit, polymorphic compare, perhaps leaving off `~cmp:` is the better choice here, since then it would work for base-0.11 and base-0.12.

Reply:
I force pushed over my commit to keep the history clean, thanks for the feedback.